src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -291,46 +291,51 @@ public class OfflineMigrationValidation
         [RavenAction("/studio-tasks/periodic-backup/test-credentials", "POST", AuthorizationStatus.ValidUser, EndpointType.Read)]
         public async Task TestPeriodicBackupCredentials()
         {
-            var type = GetQueryStringValueAndAssertIfSingleAndNotEmpty("type");
+            await TestPeriodicBackupCredentials(this);
+        }
+
+        public static async Task TestPeriodicBackupCredentials(RequestHandler requestHandler)
```

Comment:
why not wrap it up in a single processor?

Reply:
done